dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md

```diff
@@ -0,0 +1,211 @@
+./gradlew assemble :com.ibm.websphere.appserver.features:releaseNeeded
+```
+
+For more detailed instructions on building Open Liberty please see the [Building Open Liberty Wiki](https://github.com/OpenLiberty/open-liberty/wiki/Building-Open-Liberty)
+
+### Run the TCK
+
+Finally, you can run the TCK using the `buildandrun` gradle task
+
+```sh
+./gradlew io.openliberty.jakarta.concurrency.3.0_fat_tck:buildandrun
+```
+
+NOTE: Running the TCK using this method will use a published TCK version, and not the `SNAPSHOT` version from a local build. 
+
+### View the results
+
+The test results will be located under the following directory:
+
+```txt
+build/libs/autoFVT/results/junit
+```
+
+## Running the TCK for Verification
+
+> 🛠 This is still under development a beta version of the `concurrency-3.0` feature has not yet been released
+> These steps will not work until after the `concurrency-3.0` feature has a beta release.
+
+### Background
+
+1. `build.gradle` 
+   1. This project is setup using gradle.
+   2. The `build.gradle` file has tasks to setup a `wlp` directory with an Open Liberty install.
+   3. There are additional tasks that will copy dependencies, configurations, and create a server.
+2. `publish/servers`
+   1. This is where the server configuration files are located.
+   2. These files will be copied to `wlp/usr/servers/ConcurrentTCKServer` at build time.
+   3. A special `logging.properties` file is also here to ensure that the TCK logs are formatted and saved to a specific location.
+3. `publish/tckRunner`
+   1. This is a maven project that will actually run the TCK. 
+   2. This project contains the TCK configuration files that will be copied to the `wlp` directory at build time.
+   3. This includes the pom.xml that is used to get the TCK itself, and the dependencies to run the TCK. 
+   4. This is also where the TestNG and Arquillian configuration files are located.
+4. `fat/src`
+   1. This asset is ignored for this test strategy.
+
+### Project set up
+
+First, check out the release branch: 
+
+```sh
+git checkout release
+git pull
+```
+
+To make things simpler this project contains custom gradle tasks for setting up a standalone Liberty Profile.
+
+First open `build.gradle`. 
+You need to complete the `TODO` under dependencies.
+Provide a WebSphere Liberty Profile (WLP) dependency that corresponds to the release you want to test against.
+
+Then, run the `buildWLP` task
+
+```sh
+./gradlew io.openliberty.jakarta.concurrency.3.0_fat_tck:buildWLP
+```
+
+This will create a `wlp` directory within this project.
+Navigate to this directory and start your server:
+
+```sh
+cd io.openliberty.jakarta.concurrency.3.0_fat_tck/wlp
+
+./bin/server start ConcurrentTCKServer \
+-Denv.tck_username=arquillian \
+-Denv.tck_password=arquillianPassword \
+-Denv.tck_port=9080 \
+-Denv.tck_port_secure=9443
+```
+### Run the TCK
+
+Now you can run the TCK tests. 
+By default, the TCK will run against the Jakarta API and TCK version `3.0.0`.
```

Comment:
Should include an example command to run the TCK with the default 3.0.0 version here.